src/db/collections/block.rs

```diff
@@ -132,64 +131,47 @@ pub struct IncludedBlockResult {
 
 #[derive(Deserialize, Debug, Clone)]
 pub struct IncludedBlockMetadataResult {
+    #[serde(rename = "_id")]
```

Comment:
What about `IncludedBlockResult`? Should we rename there as well for consistency?